src/pages/components/IconButton.qml

```diff
@@ -32,5 +33,9 @@ IconButton {
         }
     }
 
+    onPressed: DragFilter.begin(mouse.x, mouse.y)
+    onCanceled: DragFilter.end()
+    DragFilter.screenMargin: Theme.paddingMedium
```

Comment:
Not related to this but noticed while testing that tab icon and menu icon do not highlight when tapping near the bottom corner.